app/controllers/api/messages_controller.rb

```diff
@@ -0,0 +1,8 @@
+module Api
+  class API::MessagesController < ApplicationController
```

Comment:
- [ ] This definition is incorrect. It should be the following based on your route definition:

  ```
  module Api
    class MessagesController < ApplicationController
      ...
  ```

Reply:
done